dbms/src/Interpreters/Settings.h

```diff
@@ -304,7 +304,8 @@ struct Settings
     M(SettingBool, enable_planner, true, "Enable planner")                                                                                                                                                                              \
     M(SettingBool, enable_pipeline, false, "Enable pipeline model")                                                                                                                                                                     \
     M(SettingUInt64, pipeline_task_thread_pool_size, 0, "The size of task thread pool. 0 means using number_of_logical_cpu_cores.") \
-    M(SettingUInt64, local_tunnel_version, 1, "1: not refined, 2: refined")
+    M(SettingUInt64, local_tunnel_version, 1, "1: not refined, 2: refined") \
+    M(SettingUInt64, shared_block_schemas_size, 10000, "the size of shared_block_schema")
```

Comment:
```suggestion
    M(SettingUInt64, schema_cache_size, 10000, "the cache size of shared schema")
```

Reply:
I think naming it "schema_cache_size" and use `config().getInt("schema_cache_size", 10000)` to set it in `Server.cpp` is better than add it to `Settings`. As it is

- not reloadable settings after startup
- kind of similar to "minmax_cache_size" and "mark_cache_size"